test-e2e/android/.gitignore

```diff
@@ -0,0 +1,15 @@
+*.iml
+.gradle
+/local.properties
+/.idea/caches
```

Comment:
I think the whole `/.idea` can be ignored 
e.g. https://github.com/zkmopro/mopro-android/blob/main/Example/.gitignore